spec/system/admin/products_v3/products_spec.rb

```diff
@@ -102,7 +207,39 @@
         search_for "searchable product"
 
         expect(page).to have_field "search_term", with: "searchable product"
-        # expect(page).to have_content "1 product found for your search criteria."
+        expect(page).to have_content "1 products found for your search criteria."
```

Comment:
This was commented out as "pending", because I thought the translation should be singular "product", not "products". But at least something is specified with actual code now 👍 
We can update it when we fix it.

Reply:
I agree. I just thought it would provide more value to have the assertion on this search feedback, although it's not perfect yet. I can open an issue, maybe it could even be a good fist issue?